tda/mobile_native/dotnet-maui-ios/test_checkout_dotnet_maui_ios.py

```diff
@@ -0,0 +1,27 @@
+import time
+import sentry_sdk
+import pytest
+from appium.webdriver.common.appiumby import AppiumBy
+
+@pytest.mark.skip(reason="Add to Cart stopped working on iOS")
+def test_checkout_dotnet_maui_ios(ios_dotnet_maui_driver):
+    driver = ios_dotnet_maui_driver
+
+    try:
+        driver.find_element(AppiumBy.ACCESSIBILITY_ID, "Real_World_Sample").click()
```

Comment:
Renamed:
https://github.com/sentry-demos/dotnet-maui/blob/16e03f7016b8d8355a01c56f8b14acc6c639118f/MainPage.xaml#L29

```suggestion
        driver.find_element(AppiumBy.ACCESSIBILITY_ID, "Shop_For_Plants").click()
```